packages/network-controller/src/constants.ts

```diff
@@ -26,3 +26,8 @@ export enum NetworkStatus {
 }
 
 export const INFURA_BLOCKED_KEY = 'countryBlocked';
+
+/**
+ * The default endpoint URL to hit when Infura is down.
+ */
+export const DEFAULT_INFURA_FAILOVER_URL = 'https://quicknode.com';
```

Comment:
FYI, I'm not sure whether this is the correct URL yet. I just put this in as a placeholder for now but I'm reaching out to determine what the right one is.

Reply:
Decided not to set a default failover URL across the board; the clients can do that in a migration.